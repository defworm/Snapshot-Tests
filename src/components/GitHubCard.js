import Button from "react-bootstrap/Button";
import Card from "react-bootstrap/Card";
import React from 'react'

function GitHubCard() {
    return (
        <Card style={{ width: '12rem' }}>
            <Card.Img variant='top' src='https://avatars.githubusercontent.com/u/101788833?s=400&v=4' />
            <Card.Body>
                <Card.Title>GitHubCard</Card.Title>
                <Card.Text>
                    Just a little preliminary gobblyguck to get started
                </Card.Text>
                <Button variant='primary'>Going Somewhere?</Button>
            </Card.Body>
        </Card>
    );
}

export default GitHubCard;