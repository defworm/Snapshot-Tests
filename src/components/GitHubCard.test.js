import GitHubCard from "./GitHubCard";
import renderer from 'react-test-renderer';

test('renders snapshot', () => {
    const tree = renderer.create(<GitHubCard />).toJSON()
    expect(tree).toMatchSnapshot()
})