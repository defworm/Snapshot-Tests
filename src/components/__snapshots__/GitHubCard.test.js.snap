// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders snapshot 1`] = `
<div
  className="card"
  style={
    Object {
      "width": "12rem",
    }
  }
>
  <img
    className="card-img-top"
    src="https://avatars.githubusercontent.com/u/101788833?s=400&v=4"
  />
  <div
    className="card-body"
  >
    <div
      className="card-title h5"
    >
      GitHubCard
    </div>
    <p
      className="card-text"
    >
      Just a little preliminary gobblyguck to get started
    </p>
    <button
      className="btn btn-primary"
      disabled={false}
      type="button"
    >
      Going Somewhere?
    </button>
  </div>
</div>
`;
